plop-templates/package/.gitignore

```diff
@@ -0,0 +1,18 @@
+node_modules
```

Comment:
add the 2 new entries from the rollup PR: 
`stats.html`
`rollup.bundleAnalysis.json`